supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -56,8 +56,14 @@ object JesAsyncBackendJobExecutionActor {
   val JesUnexpectedTermination = 13
   val JesPreemption = 14
 
-  def StandardException(errorCode: Status, message: String, jobTag: String) = {
-    new Exception(s"Task $jobTag failed. JES error code ${errorCode.getCode.value}. $message")
+  def StandardException(errorCode: Status, message: String, jobTag: String, returnCodeOption: Option[Int]) = {
+    val returnCodeMessage = returnCodeOption match {
+      case Some(returnCode) if returnCode == 0 => " Job exited without an error, exit code 0."
+      case Some(returnCode) => s" Job exit code $returnCode. Check the stderr for more information."
```

Comment:
ToL : If you have access to jobPaths we could also put the path of stderr in the message